src/search/search-routing.module.ts

```diff
@@ -0,0 +1,24 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+import { SearchFormComponent } from './containers';
+import { NoResultsComponent } from './containers/no-results/no-results.component';
+
+const routes: Routes = [
+  {
+    path: '',
+    component: SearchFormComponent,
+  },
+  {
+    path: 'noresults',
+    component: NoResultsComponent,
+    data: {
+      title: 'No results'
```

Comment:
@johnbenjamin-hmcts - I think the title should be `Search cases | No results` because this makes it clearer for screen readers for accessibility (so that the user knows what the "no results" is for)